tests/includes/wait-for.sh

```diff
@@ -331,3 +331,41 @@ wait_for_storage() {
 		sleep "${SHORT_TIMEOUT}"
 	fi
 }
+
+# wait_for_aws_ingress_cidrs_for_port_range blocks until the expected CIDRs
+# are present in the AWS security group rules for the specified port range.
+wait_for_aws_ingress_cidrs_for_port_range() {
+	local from_port to_port exp_cidrs cidr_type
+
+	from_port=${1}
+	to_port=${2}
+	exp_cidrs=${3}
+	cidr_type=${4}
+
+	ipV6Suffix=""
+	if [ "$cidr_type" = "ipv6" ]; then
+		ipV6Suffix="v6"
+	fi
+
+	attempt=0
+	# shellcheck disable=SC2046,SC2143
+	until [ "$attempt" != "7" ]; do
```

Comment:
also, I forgot you need to include the condition there `got_cidrs -eq exp_cidrs`